AmericanMedicalAssociation/ama-style-guide-2
# EWL-8796 trending block style guide

## Ticket(s)
**Do not** submit a Pull Request without a relevant JIRA ticket or Github issue! If you are creating a new pattern or feature, create an issue describing the need for this feature in Github **first** and then link to it below.

**Github Issue**
- N/A

**Jira Ticket**
- [EWL-8796: Add Trending block to the style guide](https://issues.ama-assn.org/browse/EWL-8796)

## Description
Added vertical and horizontal trending blocks in style guide including homepage variations.


## To Test
- with latest run lando gulp serve.
- confirm Trending Block and Trending Block as Horizontal appear under /Organisms/Trending-Block
- confirm Homepage with Trending and Homepage with Trending Vertical appear under /Pages/Homepage
- confirm block and page designs align the with following designs:

Desktop (horizontal placement): https://zpl.io/VDPKRqw
Desktop (vertical placement): https://zpl.io/bJqpz7J
Tablet (links at page width): https://zpl.io/bLOW9eB 
Mobile: https://zpl.io/2ZJDpnw

## Visual Regressions
- N/A

## Relevant Screenshots/GIFs
- N/A

## Remaining Tasks
- N/A

## Additional Notes
Anything more to add? Good things to call out here are:
- N/A

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
